recipes/hande/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "hande" %}
+{% set version = "1.5" %}
+
+{% set mpi = "openmpi" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hande-qmc/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 088bf230b921351404a6bb955814144fafd1a0298c8c23faa416ea17c06e2393
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - {{ mpi }}-mpifort
+    - cmake >=3.6
+    - make
+  host:
+    - {{ mpi }}
+    - libblas
+    - liblapack
+    - scalapack
+    - libgomp
+    - lua >=5.3
+    - python
+    - hdf5
+    - libuuid
+  run:
+    - {{ mpi }}
+    - scalapack
+    - python
+    - pandas >=0.14.1
+    - statsmodels >=0.11.0
+    - hdf5
+
+test:
+  commands:
+    - test -f ${PREFIX}/bin/hande.cmake.x
+    - test -f ${PREFIX}/lib64/libhande.a
```

Comment:
Can you please build the shared library here instead?

Reply:
Thanks, I'd be happy to but I'm not sure there is a way to do that?
At least in the docs I only see the static one https://hande.readthedocs.io/en/latest/manual/compile-with-cmake.html#installation

Reply:
This test still passes, that is not a good sign :(

Reply:
Yeah, it seems like ` cmake . --target install`  still only installs that library for static linking ([logs](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=453964&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1649)):
``` 
Install the project...
-- Install configuration: "Release"
-- Installing: $PREFIX/lib64/libhande.a
-- Installing: $PREFIX/bin/hande.cmake.x
```